tests/test_partition.py

```diff
@@ -221,10 +265,11 @@ def test_main():
     })
 
     res = client.distance_matrix(
-        origins=ORIGS,
-        destinations=DESTS,
+        origins=LOCS,
```

Comment:
Why do we want both to be the same?

Reply:
No reason, arbitrary values, this is just a rate limiter / retry test currently.